CONTRIBUTING.md

```diff
@@ -72,38 +72,23 @@ setup.py  # Project is pip-installable, project requirements managed here.
    cd weather-tools
    pip install -e ."[dev]"
    ```
-   > Note: If the execution of `pip install -e ."[dev]"` command takes much time, try upgrading your pip version:
-   > `pip install --upgrade pip`.
+    > Note: If the execution of `pip install -e ."[dev]"` command takes much time, try upgrading your pip version.
 
-3. Install `gcloud`, the Google Cloud CLI, following these [instructions](https://cloud.google.com/sdk/docs/install).
+1. Install `gcloud`, the Google Cloud CLI, following these [instructions](https://cloud.google.com/sdk/docs/install).
 
-4. Acquire adequate permissions for the project.
+1. Acquire adequate permissions for the project.
     * Run `gcloud auth application-default login`.
 
     * Make sure your account has *write* permissions to the storage bucket as well as the permissions to create a
       Dataflow job.
 
     * Make sure that both Dataflow and Cloud Storage are enabled on your Google Cloud Platform project.
 
-5. Install ecCodes and MetView on to your local environment (Anaconda seems to be the only supported way).
```

Comment:
I think your local version may be a bit out of date. Can you pull in the latest changes from the main branch?

Reply:
Performed `git rebase` and resolved conflict errors.